### DISCUSSION TITLE
Possibility to see ATS Version in Validation Report

### DISCUSSION THREAD

**Discussion by manilly** [INSPIRE-MIF/helpdesk-validator#1215] [Q&A]
Hi,
I have a question regarding the INSPIRE Validator: Is there a possibility to see which ATS is used in a test report? e.g. if I store a test report as HTML and want to see later, which ATS is used in the Test.
This would be an interesting information.
Thanks,
Manuel

**Comment by fabiovinci** [comment]
Dear @manilly,

we have decided to include links in the test reports that point to the master branch of the ATS repositories, which contain the latest version of the ATS. This approach simplifies the process of updating the associated ETSs.

Starting this year, we will only release one version of the validator per year, making it less likely that you will need to reference a test report that is more than a year old.
However, older versions of the ATS will still be accessible via their respective branches, as the branch names correspond to the TG version.

For example, see the WMS below:

![Image](https://github.com/user-attachments/assets/0a298978-c1bd-41e6-8ac4-8bcb27dd9b07)

**Reply by fabiovinci** [reply]
Dear @manilly,
did the comment above address your question?
